plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ImportNativeAppEngineStandardProjectTest.java

```diff
@@ -71,8 +71,9 @@ public void testImportAppEngineStandardJava7_from1_3_1() throws IOException, Cor
     project = SwtBotAppEngineActions.importNativeProject(bot, "AESv7", tempFolder.getRoot());
     assertTrue(project.exists());
 
-    updateOldContainers(project);
-
+    IStatus updateStatus = updateContainers(project);
```

Comment:
So there's two situations that could occur: there is no obsolete-runtime error, or there are additional errors other than the obsolete-runtime error.

Reply:
OK, added the test.